dolphinscheduler-task-plugin/dolphinscheduler-task-flink-stream/src/test/java/org/apache/dolphinscheduler/plugin/task/flink/FlinkArgsUtilsTest.java

```diff
@@ -17,116 +17,86 @@
 
 package org.apache.dolphinscheduler.plugin.task.flink;
 
-import org.apache.dolphinscheduler.plugin.task.api.TaskExecutionContext;
-import org.apache.dolphinscheduler.plugin.task.api.model.ResourceInfo;
+import org.apache.dolphinscheduler.plugin.task.api.TaskConstants;
+import org.apache.dolphinscheduler.plugin.task.flink.entity.ParamsInfo;
+import org.apache.dolphinscheduler.plugin.task.flink.enums.FlinkStreamDeployMode;
 import org.junit.Assert;
 import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
-import java.util.List;
+import java.util.Properties;
 
 public class FlinkArgsUtilsTest {
 
-    private String joinStringListWithSpace(List<String> stringList) {
```

Comment:
Maybe we should keep the old test for flink task, and add the new one for flink-stream task, WDYT?

Reply:
But I have changed some parameters, and the original UT is outdated. @caishunfeng